### DISCUSSION TITLE
Seed coordinates in tesselation file

### DISCUSSION THREAD

**Discussion by mustafasubasic** [neperfepx/neper#661] [Q&A] [answered]
Dear all

Are the seed coordinates in the tesselation file (.tess) given as x,y,z-coordinates and weight the coordinate point in the middle of a grain? I have tested creating a model with just a few grains but the coordinates doesn't make sense to me.

Thank you!

Best regards
Mustafa

**Comment by rquey** [comment, marked as answer]
No, the seed coordinates and the cell centroids (geometrical centers) are different things. To get the centroids, use `-statcell x,y,z`.

**Reply by mustafasubasic** [reply]
Thank you very much!
